src/util/errors.ts

```diff
@@ -64,4 +64,10 @@ export class NotTextChannelError extends Error {
     public constructor() {
         super("Expected a text channel but got another type");
     }
+}
+
+export class TenorError extends Error {
+    public constructor() {
+        super("Tenor is unavailable or the request is malformed");
```

Comment:
A new error type for when Tenor is unavailable or the request is malformed due to an API change